PASS __tests__/App-test.tsx
  âœ“ renders correctly (79 ms)

Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        1.509 s, estimated 2 s
Ran all test suites.

ReferenceError: You are trying to `import` a file after the Jest environment has been torn down. From __tests__/App-test.tsx.

      at Object.get BackHandler [as BackHandler] (node_modules/react-native/index.js:176:12)
      at node_modules/@react-navigation/native/lib/commonjs/useBackButton.js:64:37
      at invokePassiveEffectCreate (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12009:24)
      at Object.invokeGuardedCallbackProd (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:9492:14)
      at invokeGuardedCallback (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:9596:35)

ReferenceError: You are trying to `import` a file after the Jest environment has been torn down. From __tests__/App-test.tsx.

      at Object.get Linking [as Linking] (node_modules/react-native/index.js:226:12)
      at subscribe (node_modules/@react-navigation/native/lib/commonjs/useLinking.native.tsx:109:5)
      at node_modules/@react-navigation/native/lib/commonjs/useLinking.native.js:218:12
      at invokePassiveEffectCreate (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12009:24)
      at Object.invokeGuardedCallbackProd (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:9492:14)
/home/nom4d/delivery/GuardianDock/node_modules/scheduler/cjs/scheduler.development.js:55
            throw e;
            ^

TypeError: _reactNative.BackHandler.addEventListener is not a function
    at /home/nom4d/delivery/GuardianDock/node_modules/@react-navigation/native/lib/commonjs/useBackButton.js:64:49
    at invokePassiveEffectCreate (/home/nom4d/delivery/GuardianDock/node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12009:24)
    at Object.invokeGuardedCallbackProd (/home/nom4d/delivery/GuardianDock/node_modules/react-test-renderer/cjs/react-test-renderer.development.js:9492:14)
    at invokeGuardedCallback (/home/nom4d/delivery/GuardianDock/node_modules/react-test-renderer/cjs/react-test-renderer.development.js:9596:35)
    at flushPassiveEffectsImpl (/home/nom4d/delivery/GuardianDock/node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12083:13)
    at unstable_runWithPriority (/home/nom4d/delivery/GuardianDock/node_modules/scheduler/cjs/scheduler.development.js:414:16)
    at runWithPriority (/home/nom4d/delivery/GuardianDock/node_modules/react-test-renderer/cjs/react-test-renderer.development.js:2124:14)
    at flushPassiveEffects (/home/nom4d/delivery/GuardianDock/node_modules/react-test-renderer/cjs/react-test-renderer.development.js:11968:18)
    at /home/nom4d/delivery/GuardianDock/node_modules/react-test-renderer/cjs/react-test-renderer.development.js:11864:15
    at workLoop (/home/nom4d/delivery/GuardianDock/node_modules/scheduler/cjs/scheduler.development.js:365:38)

Node.js v17.6.0
error Command failed with exit code 1.
